### DISCUSSION TITLE
Github Education Application Approved but still not get benefits

### DISCUSSION THREAD

**Discussion by iamjunaidjutt** [community/community#123021] [GitHub Education]
### Hi everyone,

![Screenshot 2024-05-09 002528](https://github.com/community/community/assets/84816593/46d71050-c27c-4dcb-beca-e9dff9aeb9c3)
![Screenshot 2024-05-09 002541](https://github.com/community/community/assets/84816593/bbaae545-1e9d-4e11-96af-bdb42dd96785)

**Comment by iamjunaidjutt** [comment]
I don't know what can I do now because 4 days have passed but still benefits are not available even not a single response found from GitHub Education

**Comment by davevad93** [comment]
Hi @iamjunaidjutt ,

When your student benefits will be finally available you will see the following message:

`Congrats! Your request for @iamjunaidjutt was approved and your benefits are now available.`

It usually takes 1-2 business days to receive the benefits, sometimes it can take a bit more. Wait another couple of days and if your benefits aren't available after that time, contact [GitHub Education Support](https://support.github.com/request/education).
